servers/admin_ui/pages/env_vars_editor.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+import streamlit as st
+
+from agent_guard_core.credentials.aws_secrets_manager_provider import AWSSecretsProvider
+from agent_guard_core.credentials.secrets_provider import SecretProviderException
+from servers.admin_ui.common import get_secret_provider, get_secret_provider_name, print_header
+
+# get server configuration
+secrets_dictionary = None
+try:
+    secret_provider = get_secret_provider()
+    secrets_dictionary = secret_provider.get_secret_dictionary()
+
+except SecretProviderException as e:  # Fixed typo in exception handling
```

Comment:
Are all these comments in this file needed? I would remove them and consolidate the lines, it would make the code easier to read in my opinion